xml/docker_building_images.xml

```diff
@@ -8,68 +8,74 @@
-   application you want to run inside a &docker; container, see
-   <xref linkend="docker.dockerizing.application"/>.
-  </para>
- </note>
- <!-- This section is commented out after discussion with Federica as KIWI
- should not be used for building base images outside SUSE. Even though it is
- possible, the only supported way is to activate pre-built images of SLES -->
- <!--
- <sect1 xml:id="Building_Base_Images_KIWI">
-  <title>Building Base &slsa; Images with KIWI</title>
-  <para>
-   KIWI is a command line tool that enables you to build images in a
-   variety of formats. You can use the tool to build a base &docker; image of
-   &slsa;. For more details about KIWI installation and usage, please refer
-   to the
-   <link xlink:href="https://doc.opensuse.org/projects/kiwi/doc/#chap.introduction">KIWI documentation</link>.
-  </para>
+ <sect1 xml:id="Downloading_Pre-build_Images">
+  <title>Downloading Base &slsa; Images</title>
   <para>
-   The procedure of building &docker; images is given in the
-   <link xlink:href="https://doc.opensuse.org/projects/kiwi/doc/#chap.docker">Docker Images chapter</link>.
+   How to obtain a pre-built base image depends on its &productname; version:
   </para>
- </sect1>
- -->
- <sect1 xml:id="Building_Pre-build_Images">
-  <title>Obtaining Base &slsa; Images</title>
-  <para>
-   To obtain the base &slsa; images from &suse; registry, use the following
-   command:</para>
-  <screen>&prompt.user;docker pull registry.suse.com/suse/<replaceable>IMAGENAME</replaceable></screen>
 
-  <para>
-   For example, to get the one for &productname; 15, use:
-  </para>
- <screen>&prompt.user;docker pull registry.suse.com/suse/sle15</screen>
+  <variablelist>
+   <varlistentry>
+    <term>Up to &sle;&nbsp;12 SP2</term>
+    <listitem>
+     <screen>&prompt.sudo;zypper in <replaceable>IMAGENAME</replaceable></screen>
+     <para>
+      For example, for &productname;&nbsp;11 SP4, use
+     </para>
+     <screen>&prompt.sudo;zypper in sles11sp4</screen>
+     <para>
+      After obtaining the image, you need to activate it. Proceed as follows:
+     </para>
 
-  <para>
-   Pre-built images do not have repositories configured. But when the &docker;
-   host has a &slea; subscription that provides access to the product used in the
-   image, Zypper will automatically have access to the right repositories.
-  </para>
+     <procedure>
+      <title>Activating the Base Image for up to &slsa;&nbsp;12 SP2</title>
+      <step>
+       <para>
+        Get the proper image name with <literal>sle2docker</literal> by running
+       </para>
+       <screen>&prompt.user;sle2docker list</screen>
+      </step>
+      <step>
+       <para>
+        Activate the image by using the image name from the previous step:
+       </para>
+       <screen>&prompt.user;sle2docker activate <emphasis>PRE-BUILT_IMAGE_NAME</emphasis></screen>
+      </step>
+      <step>
+       <para>
+        Check if the image was successfully activated by running
```

Comment:
you can even skip "by running". Maybe shorten to: "Check if the image was successfully activated:"